helm_remote/Tiltfile

```diff
@@ -91,14 +93,11 @@ def helm_remote(chart, repo_url='', repo_name='', release_name='', values=[], se
     else:
         k8s_yaml(helm(chart_target, name=release_name, values=values, set=set))
 
-
 def install_crds(name, directory):
     name += '-crds'
+    files = str(local(r"grep --include '*.yaml' --include '*.yml' -rEil '\bkind\s*:\s+CustomResourceDefinition\s*$' %s || exit 0" % directory)).strip()
```

Comment:
does this not need a `command_bat` version?

Reply:
there's a great grep implementation for windows! with good install packages for scoop, chocolatey, etc

Reply:
Sure, but that would mean the end-user would have to install it separately, right? Seems less than ideal (not that I know any other acceptable windows-friendly substitute...)

Reply:
yep!

In the medium-term, there are some Tilt APIs that should make this easier to do without `local()`. In the long-term, I'd love it if Tilt was smarter about ordering these CRDs, so that we didn't need to do this at all....